### DISCUSSION TITLE
How to tell if the .NET Core Hosting Bundle is installed and which version is installed?

### DISCUSSION THREAD

**Discussion by SetTrend** [dotnet/aspnetcore#57595] [Q&A] [answered]
How can I check if the [.NET Core Hosting Bundle](https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/hosting-bundle) has already been installed to a server and where can I find the installed .NET Core Hosting Bundle version?

What happens if I update the .NET Core Hosting Bundle? Will the old version be removed or will there be running two versions in parallel?

**Comment by gfoidl** [comment]
> How can I check if the [.NET Core Hosting Bundle](https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/hosting-bundle) has already been installed to a server and where can I find the installed .NET Core Hosting Bundle version?

See [Module version and Hosting Bundle installer logs](https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/hosting-bundle?view=aspnetcore-8.0#module-version-and-hosting-bundle-installer-logs) (in the same article you linked above, just a bit scrolling down).

If that folder exists, then the bundle is installed.

> What happens if I update the .NET Core Hosting Bundle?

Updated in-place. That's why it's in the path `V2`, so it has a very high compatibility bar.
Maybe sometime in the future there may be a `V3` (but no plans AFAIK at the moment)

**Reply by SetTrend** [reply]
Ah, I see … Thanks for enlightening me! 👍 

If I'd be a consumer of a shared server (which I am for one of my websites), I wouldn't have access to these directories. Only to _IIS Manager_. Is there a way to tell if the .NET Core Hosting Bundle is installed by using IIS Manager?

**Reply by gfoidl** [reply, marked as answer]
![grafik](https://github.com/user-attachments/assets/2ec9f59a-ea7c-4207-958e-13cfa83ee704)

![grafik](https://github.com/user-attachments/assets/acb41cb8-3ece-4ac2-8abe-d6d4d4581c0f)
